app/assets/javascripts/admin/addon/templates/user-fields.hbs

```diff
@@ -8,7 +8,7 @@
       <AdminUserFieldItem
         @userField={{uf}}
         @fieldTypes={{this.fieldTypes}}
-        @firstField={{this.sortedFields.firstObject}}
```

Comment:
I'm a little confused about why these changes were made - there were no violations before, and I don't think this PR is introducing any linting rules? 🤔